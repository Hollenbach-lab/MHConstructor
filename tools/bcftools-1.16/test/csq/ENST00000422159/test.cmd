{bin}/bcftools csq -f ENST00000422159.fa -g ENST00000422159.gff multiallelic.vcf  | {bin}/bcftools query -f'[%POS\t%REF\t%ALT\t%GT\t%TBCSQ\n]' | ../sort-csq -q
echo
{bin}/bcftools csq -f ENST00000422159.fa -g short.gff multiallelic.vcf | {bin}/bcftools query -f'[%POS\t%REF\t%ALT\t%GT\t%TBCSQ\n]' | ../sort-csq -q
