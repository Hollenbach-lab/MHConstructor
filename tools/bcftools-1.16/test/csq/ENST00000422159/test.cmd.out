279	TAGAGAG	T,TAG	0|1	.	3_prime_utr|GHRL|ENST00000287656|protein_coding,3_prime_utr|GHRL|ENST00000335542|protein_coding,3_prime_utr|GHRL|ENST00000429122|protein_coding,3_prime_utr|GHRL|ENST00000430179|protein_coding,3_prime_utr|GHRL|ENST00000437422|protein_coding,3_prime_utr|GHRL|ENST00000439975|protein_coding,3_prime_utr|GHRL|ENST00000449238|protein_coding,3_prime_utr|GHRL|ENST00000457360|protein_coding,stop_lost&frameshift|GHRL|ENST00000422159|protein_coding|-|91L*>91K|279TAGAGAG>TAG,stop_lost&frameshift|GHRL|ENST00000446937|protein_coding|-|40L*>40K|279TAGAGAG>TAG
279	TAGAGAG	T,TAG	0|2	.	3_prime_utr|GHRL|ENST00000287656|protein_coding,3_prime_utr|GHRL|ENST00000335542|protein_coding,3_prime_utr|GHRL|ENST00000429122|protein_coding,3_prime_utr|GHRL|ENST00000430179|protein_coding,3_prime_utr|GHRL|ENST00000437422|protein_coding,3_prime_utr|GHRL|ENST00000439975|protein_coding,3_prime_utr|GHRL|ENST00000449238|protein_coding,3_prime_utr|GHRL|ENST00000457360|protein_coding,inframe_deletion&stop_retained|GHRL|ENST00000422159|protein_coding|-|90SL*>90*|279TAGAGAG>T,inframe_deletion&stop_retained|GHRL|ENST00000446937|protein_coding|-|39SL*>39*|279TAGAGAG>T

279	TAGAGAG	T,TAG	0|1	.	stop_lost&frameshift|GHRL|ENST00000422159|protein_coding|-|91L*>91K|279TAGAGAG>TAG
279	TAGAGAG	T,TAG	0|2	.	inframe_deletion&stop_retained|GHRL|ENST00000422159|protein_coding|-|90SL*>90*|279TAGAGAG>T
